clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -50,31 +49,29 @@
  */
 @PublicApi
 public final class FileInStream extends InputStream implements BoundedStream, Seekable {
-  /** Logger for this class */
+  /** Logger for this class. */
```

Comment:
The convention I have been following is to start sentences with an uppercase letter and end them with a period. An exception to this convention is a single sentence starting with a lowercase letter and no period at the end. It might just be easier to always use an uppercase letter and a period (I think Mesos does that). In any case, let me know if you prefer a different convention or you would like to me revert this change.